app/frontend/styles/application.scss

```
@@ -36,6 +36,8 @@ $moj-images-path: "~@ministryofjustice/frontend/moj/assets/images/";
 @import "_tag";
 @import "_back_link";
 
+@import "dfe_component/_all";
+
 // Shared (custom components)
```

Might want to have some additional commenting to describe the differences between BAT-wide and application-wide styles

```suggestion
// Shared (common components replicated across BAT services)
@import "dfe_component/_all";

// Shared (components used by all application interfaces)
```
```suggestion
@import "dfe_component/_all";

// Shared (custom components)
```